packages/block-library/src/heading/heading-level-dropdown.js

```diff
@@ -41,7 +46,19 @@ const POPOVER_PROPS = {
  *
  * @return {WPComponent} The toolbar.
  */
-export default function HeadingLevelDropdown( { selectedLevel, onChange } ) {
+export default function HeadingLevelDropdown( {
+	clientId,
+	selectedLevel,
+	onChange,
+} ) {
+	const instanceId = useInstanceId( HeadingLevelDropdown );
```

Comment:
You can provide the prefix you want in [useInstanceId](https://github.com/WordPress/gutenberg/blob/c1b05533dd1732a6f5d0bdd79080b1b671144b4a/packages/compose/src/hooks/use-instance-id/index.js#L25) and reuse it in both places below.

ex.
```
const instanceId = useInstanceId(
	HeadingLevelDropdown,
        'block-library-heading__heading-level-dropdown__help'
);
```

Reply:
But should I? While functionally, that would have the same effect, it seems a little confusing from a code readability perspective, since I'm prefixing an id referring to the entire component with a class name specific to one part of it.

Reply:
The variable name could change to `descriptionId` to better reflect its content. That's why the second param exists. Of course it works as is now too..

Reply:
Oh, good point. It might also be slightly faster to do it the way you're suggesting, so I'll go ahead and make the change.